resources/setups/qualification/site_config.json

```diff
@@ -8,7 +8,7 @@
     "categories": [ "Base", "Connection", "Security", "NTP", "DNS", "Communication", "Protocol", "PoE", "IoT"],
     "expected": [ "Required Pass", "Required Pass for PoE Devices", "Required Pass for BACnet Devices", "Required Pass for IoT Devices", "Recommended Pass" ]
   },
-  "include": "../qualification/qualification_tests.json",
+  "include": "../../resources/setups/qualification/qualification_tests.json",
```

Comment:
name is redundant, something is wrong here.  Should this be qualification/test_config.json?

The goal is to have the actual filename (.json) be semantically meaningful to describe what kind of information is in that file (test config?), while the directory path indicates which one (for qualification).

Reply:
Agree as per your previous comment on the `common/test_config.json` would be the suitable name

Thinking more, what I've done here may not appropriate or needed yet. My intent was to try and avoid repetition of tests by including from a `common/test_config.json`, but there may be valid reasons to categorise tests differently in different test environments which is what is in these files. Maybe best to have only a `baseline/test_config.json` which describes all the test for now.